src/Microsoft.Crank.JobOjectWrapper/Program.cs

```diff
@@ -30,8 +30,26 @@
 Console.WriteLine($"Filename: {process.StartInfo.FileName}");
 Console.WriteLine($"Args: {process.StartInfo.Arguments}");
 
+process.OutputDataReceived += (_, e) =>
```

Comment:
I think it's equivalent to not invoking these:
```
RedirectStandardOutput = true,
RedirectStandardError = true,
```

Reply:
These are both already being set in the StartInfo of the process starting this wrapper and the process being started by this wrapper.

Reply:
As far as I can tell, just having the redirect enabled does not cause the parent process to actually receive the output live, and we need the output live in order to tell when the application has started.

Reply:
> As far as I can tell, just having the redirect enabled does not cause the parent process to actually receive the output live

Was wondering the opposite, remove the code completely.

Reply:
Looks like that works as well, went ahead and updated to remove code rather than add code.